api/migrations/0001_initial.py

```diff
@@ -0,0 +1,61 @@
+# Generated by Django 4.2.4 on 2023-08-27 14:00
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='countryID',
+            fields=[
+                ('country_id', models.CharField(max_length=3, primary_key=True, serialize=False)),
```

Comment:
add a comment here about what the id represents since it's not an auto-generated one and has actual meaning.

Also why not use the 2 letter ISO codes instead of 3 letter ones?